pkg/recovery/securerecoverer.go

```diff
@@ -0,0 +1,35 @@
+package recovery
+
+import (
+	"os"
+	"syscall"
+)
+
+// SecureRecoverer properties
+// Reboot: does a reboot if true
+// Sync: sync file descriptors and devices
+type SecureRecoverer struct {
+	Reboot bool
+	Sync   bool
+}
+
+// Recover by reboot or poweroff without or with sync
+func (sr SecureRecoverer) Recover() error {
+	if sr.Sync {
+		for _, f := range []*os.File{
+			os.Stdout,
+			os.Stderr,
+		} {
+			syscall.Fsync(int(f.Fd()))
```

Comment:
`f.Sync()` should do the same without resorting to raw syscalls, afaict from the implementation (https://golang.org/src/os/file_posix.go?s=3352:3379#L127)

Reply:
Done